Gemfile

```diff
@@ -52,3 +52,5 @@ end
 
 # Windows does not include zoneinfo files, so bundle the tzinfo-data gem
 gem 'tzinfo-data', platforms: [:mingw, :mswin, :x64_mingw, :jruby]
+# Using HAML instead of ERB for transparent view files
+gem "haml", "~> 5.1"
```

Comment:
We could use HAML as the default template engine for scaffolding. No configuration is needed, we just need to add it here:
```suggestion
gem "haml", "~> 5.1"
# Use HAML as the default template engine for scaffolding
gem 'haml-rails', '~> 2.0', '>= 2.0.1'
```